main.go

```diff
@@ -80,6 +81,7 @@ func (g *Git) Init() *Repo {
 func (g *Git) Clone(ctx context.Context, url string) *Repo {
 	clone := g.container().
 		WithWorkdir("/tmp").
+		WithEnvVariable("CACHE_BUSTER", time.Now().Format(time.RFC3339Nano)).
```

Comment:
I wonder if this should be done on every `container()` invocation.

WDYT @shykes ?

Reply:
Good question. I think no, it would be too broad. Let's start with `clone` and `push`, then we can iterate from there. I have helper code from another module that I can reuse.